readme.md

```diff
@@ -1,70 +1,100 @@
+- Each number must be within a large clickable box.
+- Each number must be horizontally and vertically centered within their clickable box.
+- The boxes must be laid out evenly within a grid.
+- On mobile and small screen devices 1 number must be displayed in each grid row.
+- On tablet and middle sized devices 2 numbers must be displayed in each grid row.
+- On desktop and large screen devices 3 numbers must be displayed in each grid row.
+- The grid should have a maximum width so that the clickable boxes do not get too large.
+- The grid should be horizontally centered on the screen.
 
-__Constraints__: On mobile devices the grid will display 1 number per row. On tablet devices 2 numbers per row and on desktop and larger devices 3 numbers per row. The numbers should fill their rows equally. When the screen width grows very large a margin should appear either side of the grid so that the boxes don’t keep getting larger.
+### Other
 
-You do not need to use any libraries but if you do, you should be able to explain the reasons for your choices in the Readme file.
+We'd appreciate it if you could update this readme with additional information about your solution. Things to mention might include the tools used, alternative solutions considered, improvements you'd like to make, or things you'd do differently if you had more time.
 
-If there is any information that you feel is missing from the story then please make sensible decisions (for example it’s up to you how you define ‘mobile’, as long as it’s sensible, similarly please define highlighting).
+### Criteria for assessment
 
-__Criteria for assessment__: Important things that will be assessed are:
+Your completed test exercise will be marked according to a score sheet. We will focus on the following areas:
 
-* Project layout.
-* Coding style, i.e. your approach not whether you use semicolons or not ;)
-* Your approach to unit testing/TDD
-* Maintainability and extensibility
-* Use of responsive design
-* Performance, Accessibility and use of Semantic HTML
-* Naming conventions used
-* Meeting the requirements
+* That your solution meets the requirements.
+* How accessible your application is.
+* Your approach to testing the application.
+* The use of responsive design.
+* Clear project structure and conventions.
+* How readable and understandable your code is.
+* The performance of your completed application.
 
-The test won’t be assessed on design at all, so don’t worry how it looks.
+You won’t be assessed on design at all, so please don’t worry how it looks.
```

Comment:
Just a random thing unrelated to the README, and not really up to me to decide, but do we really not assess on design at all? I feel like part of the job is making things look good when you don't always have concrete designs. Obviously it's not the main factor to assess on though

Also I feel like it kind of contradicts having the 'visual requirements' in some ways because some of those could be argued to be design related (e.g. centring and picking a suitable maximum width)

Reply:
Beyond the visual criteria listed and need for elements to be highlighted, there's no mention of design at all in the [grading criteria](https://docs.google.com/document/d/1n8xWpDNkHLG8mogq4F1kneLK-fxXQoXW1NSr7gu493E/edit).